src/main/java/duke/utils/TaskList.java

```diff
@@ -0,0 +1,83 @@
+package duke.utils;
+
+import java.util.ArrayList;
+
+import duke.tasks.Task;
+
+/**
+ * This class implementions the functionality of a variable length list containg Task objects.
+ *
+ * @author delishad21
+ */
+public class TaskList {
+    private ArrayList<Task> toDoList;
```

Comment:
Perhaps taskList may be a better name, as toDo sounds like one of the task types?